include/libswiftnav/bits.h

```diff
@@ -23,5 +23,6 @@ void setbits(u8 *buff, u32 pos, u32 len, s32 data);
 void bitcopy(void *dst, u32 dst_index,
              const void *src, u32 src_index, u32 count);
 void bitshl(void *buf, u32 size, u32 shift);
+u8 get_set_bit_number(u32 v);
```

Comment:
get_set_bit_number -> get_bit_count / getbitcount ?
get_set_ prefix looks a bit confusing to my taste.